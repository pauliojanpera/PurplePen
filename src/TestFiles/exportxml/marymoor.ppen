<course-scribe-event>
  <event id="1">
    <title>Marymoor WIOL 2</title>
    <notes>This is a sample event for testing.</notes>
    <map kind="OCAD" scale="10000" ignore-missing-fonts="false">Marymoor.ocd</map>
    <all-controls print-scale="10000" description-kind="symbols" />
    <numbering start="31" disallow-invertible="true" />
    <punch-card rows="3" columns="8" left-to-right="true" top-to-bottom="false" />
    <course-appearance />
    <descriptions lang="en" />
  </event>
  <control id="1" kind="start">
    <location x="56.8" y="-8.7" />
    <description box="D" iof-2004-ref="4.4" />
  </control>
  <control id="2" kind="finish">
    <location x="53.2" y="-2.8" />
    <description box="all" iof-2004-ref="14.1" />
  </control>
  <control id="35" kind="normal">
    <code>35</code>
    <location x="115.6" y="-11.8" />
    <description box="C" iof-2004-ref="0.1E" />
    <description box="D" iof-2004-ref="4.5" />
    <description box="G" iof-2004-ref="11.2E" />
    <punch-pattern size="7">
1000000
0000000
0000000
0001000
0000000
0000000
0000001
</punch-pattern>
  </control>
  <control id="36" kind="normal">
    <code>36</code>
    <location x="128.4" y="6.1" />
    <description box="D" iof-2004-ref="4.9" />
    <punch-pattern size="7">
1001001
0000000
0000000
1001001
0000000
0000000
1001001
</punch-pattern>
  </control>
  <control id="37" kind="normal">
    <code>37</code>
    <location x="121.8" y="-23" />
    <description box="D" iof-2004-ref="4.5" />
    <description box="G" iof-2004-ref="11.6NW" />
    <punch-pattern size="7">
1000000
0000000
0000000
1000000
0000000
0000000
1001001
</punch-pattern>
  </control>
  <control id="38" kind="normal">
    <code>38</code>
    <location x="50.3" y="2.9" />
    <description box="D" iof-2004-ref="4.3" />
    <description box="G" iof-2004-ref="11.5E" />
    <punch-pattern size="7">
0100010
0000000
0000000
0010100
0000000
0000000
0001000
</punch-pattern>
  </control>
  <control id="39" kind="normal">
    <code>39</code>
    <location x="129.2" y="37.7" />
    <description box="D" iof-2004-ref="4.8" />
    <description box="G" iof-2004-ref="11.6S" />
    <punch-pattern size="7">
0010100
0000000
0000000
1000001
0000000
0000000
0010100
</punch-pattern>
  </control>
  <control id="40" kind="normal">
    <code>40</code>
    <location x="124.2" y="29.5" />
    <description box="D" iof-2004-ref="1.9" />
    <description box="G" iof-2004-ref="11.3W" />
    <punch-pattern size="7">
1000001
0000000
0000000
1000001
0000000
0000000
1001001
</punch-pattern>
  </control>
  <control id="41" kind="normal">
    <code>41</code>
    <location x="28" y="6.2" />
    <description box="D" iof-2004-ref="4.5" />
    <description box="G" iof-2004-ref="11.2NW" />
    <punch-pattern size="7">
0101000
0000010
0000000
0101000
0000000
0000000
0100000
</punch-pattern>
  </control>
  <control id="42" kind="normal">
    <code>42</code>
    <location x="38.4" y="-14" />
    <description box="D" iof-2004-ref="4.7" />
    <description box="E" iof-2004-ref="4.7" />
    <description box="F" iof-2004-ref="10.2" />
    <punch-pattern size="7">
1000001
0000000
0000000
0001000
0000000
0000000
1000001
</punch-pattern>
  </control>
  <control id="43" kind="normal">
    <code>43</code>
    <location x="84" y="34.5" />
    <description box="C" iof-2004-ref="0.1E" />
    <description box="D" iof-2004-ref="3.5" />
    <description box="E" iof-2004-ref="4.5" />
    <description box="F" iof-2004-ref="10.1" />
    <punch-pattern size="7">
1001001
0000000
0000000
1001001
0000000
0000000
1001001
</punch-pattern>
  </control>
  <control id="44" kind="normal">
    <code>44</code>
    <location x="37.6" y="-22.8" />
    <description box="D" iof-2004-ref="6.1" />
    <description box="G" iof-2004-ref="11.1E" />
    <punch-pattern size="7">
1001001
0000000
0000000
1001001
0000000
0000000
1001001
</punch-pattern>
  </control>
  <control id="45" kind="normal">
    <code>45</code>
    <location x="55.9" y="25.6" />
    <description box="D" iof-2004-ref="4.5" />
    <description box="G" iof-2004-ref="11.6SW" />
    <punch-pattern size="7">
0000000
1001001
0000000
0000000
1001001
0000000
0000000
</punch-pattern>
  </control>
  <control id="46" kind="normal">
    <code>46</code>
    <location x="-9.2" y="8.3" />
    <description box="D" iof-2004-ref="1.12" />
    <description box="G" iof-2004-ref="11.1NW" />
    <punch-pattern size="7">
0000001
0000000
0000000
0001001
0000000
0000000
1001001
</punch-pattern>
  </control>
  <control id="47" kind="normal">
    <code>47</code>
    <location x="2.51" y="31.32" />
    <description box="D" iof-2004-ref="4.5" />
    <description box="G" iof-2004-ref="11.1NW" />
    <punch-pattern size="7">
0001000
0000000
0000000
0001000
0000000
0000000
0001000
</punch-pattern>
  </control>
  <control id="48" kind="normal">
    <code>48</code>
    <location x="21.5" y="40.2" />
    <description box="D" iof-2004-ref="1.9" />
    <description box="G" iof-2004-ref="11.3NE" />
    <punch-pattern size="7">
1001001
0000000
0000000
1001001
0000000
0000000
1001001
</punch-pattern>
  </control>
  <control id="49" kind="normal">
    <code>49</code>
    <location x="22" y="-20.8" />
    <description box="D" iof-2004-ref="5.1" />
    <description box="G" iof-2004-ref="11.8SW" />
    <punch-pattern size="7">
1000000
0000000
0000000
0001000
0000000
0000000
0000001
</punch-pattern>
  </control>
  <control id="50" kind="normal">
    <code>50</code>
    <location x="45.2" y="22.8" />
    <description box="D" iof-2004-ref="1.12" />
    <description box="G" iof-2004-ref="11.3NW" />
    <punch-pattern size="7">
1010000
0001000
1010000
0000000
1000000
0000000
0000000
</punch-pattern>
  </control>
  <control id="51" kind="normal">
    <code>51</code>
    <location x="16.4" y="-11.9" />
    <description box="D" iof-2004-ref="4.5" />
    <description box="G" iof-2004-ref="11.1NW" />
    <punch-pattern size="7">
1001001
0000000
0000000
0001000
0000000
0000000
0001000
</punch-pattern>
  </control>
  <control id="52" kind="normal">
    <code>52</code>
    <location x="43" y="-11.1" />
    <description box="D" iof-2004-ref="1.9" />
    <description box="G" iof-2004-ref="11.5W" />
    <punch-pattern size="7">
0000001
0000000
0000000
0001000
0000000
0000000
1000000
</punch-pattern>
  </control>
  <control id="53" kind="normal">
    <code>53</code>
    <location x="25.8" y="-10.5" />
    <description box="D" iof-2004-ref="4.9" />
    <description box="G" iof-2004-ref="11.1SW" />
    <punch-pattern size="7">
0000000
0001000
0000000
0100010
0000000
0000000
0100010
</punch-pattern>
  </control>
  <control id="54" kind="normal">
    <code>54</code>
    <location x="53.7" y="8.2" />
    <description box="D" iof-2004-ref="3.5" />
    <description box="E" iof-2004-ref="4.5" />
    <description box="F" iof-2004-ref="10.1" />
    <description box="G" iof-2004-ref="11.2NW" />
    <punch-pattern size="7">
1000000
0000000
0000000
1000000
0000000
0000000
1001001
</punch-pattern>
  </control>
  <control id="55" kind="normal">
    <code>55</code>
    <location x="51.2" y="10.5" />
    <description box="D" iof-2004-ref="2.4" />
    <punch-pattern size="7">
0001000
0000010
0100000
0000000
0100000
0000010
0001000
</punch-pattern>
  </control>
  <control id="56" kind="normal">
    <code>56</code>
    <location x="81.3" y="24" />
    <description box="C" iof-2004-ref="0.1W" />
    <description box="D" iof-2004-ref="4.5" />
    <description box="G" iof-2004-ref="11.1E" />
    <punch-pattern size="7">
0001000
0000000
0000000
0001000
0000000
0000000
0001000
</punch-pattern>
  </control>
  <control id="57" kind="normal">
    <code>57</code>
    <location x="90" y="19.3" />
    <description box="D" iof-2004-ref="4.5" />
    <description box="G" iof-2004-ref="11.1NW" />
    <punch-pattern size="7">
0000001
0000000
0000000
0001001
0000000
0000000
1001001
</punch-pattern>
  </control>
  <control id="58" kind="normal">
    <code>58</code>
    <location x="20.3" y="-4" />
    <description box="D" iof-2004-ref="5.2" />
    <description box="E" iof-2004-ref="5.1" />
    <description box="F" iof-2004-ref="10.2" />
    <punch-pattern size="7">
0000000
0100010
0000000
0000000
0000000
0100010
0000000
</punch-pattern>
  </control>
  <control id="59" kind="normal">
    <code>59</code>
    <location x="51.2" y="-27" />
    <description box="D" iof-2004-ref="4.5" />
    <description box="G" iof-2004-ref="11.1N" />
    <punch-pattern size="7">
0100010
0000000
0000000
0100010
0000000
0000000
0001000
</punch-pattern>
  </control>
  <control id="70" kind="normal">
    <code>70</code>
    <location x="21.3" y="11.8" />
    <description box="D" iof-2004-ref="3.4" />
    <description box="G" iof-2004-ref="11.8NE" />
    <punch-pattern size="7">
0000000
0000000
1001001
0000000
0000000
1001001
0000000
</punch-pattern>
  </control>
  <control id="71" kind="normal">
    <code>71</code>
    <location x="40.1" y="17.4" />
    <description box="D" iof-2004-ref="4.9" />
    <description box="G" iof-2004-ref="11.1S" />
    <punch-pattern size="7">
1001001
0000000
0000000
0001000
0000000
0000000
1001001
</punch-pattern>
  </control>
  <control id="72" kind="normal">
    <code>72</code>
    <location x="-0.7" y="10.3" />
    <description box="D" iof-2004-ref="5.9" />
    <description box="G" iof-2004-ref="11.5NW" />
    <punch-pattern size="7">
1001001
0000000
0000000
1000001
0000000
0000000
1001001
</punch-pattern>
  </control>
  <control id="73" kind="normal">
    <code>73</code>
    <location x="57.4" y="-13.8" />
    <description box="C" iof-2004-ref="0.1N" />
    <description box="D" iof-2004-ref="4.9" />
    <description box="G" iof-2004-ref="11.1W" />
    <punch-pattern size="7">
0100010
0000000
0000000
0010100
0000000
0000000
0001000
</punch-pattern>
  </control>
  <control id="74" kind="normal">
    <code>74</code>
    <location x="61.2" y="-18.8" />
    <description box="C" iof-2004-ref="0.1S" />
    <description box="D" iof-2004-ref="4.9" />
    <description box="G" iof-2004-ref="11.1SW" />
    <punch-pattern size="7">
1001000
0000000
0000000
1001001
0000000
0000000
0001001
</punch-pattern>
  </control>
  <control id="75" kind="normal">
    <code>75</code>
    <location x="72" y="27" />
    <description box="D" iof-2004-ref="5.2" />
    <description box="E" iof-2004-ref="3.5" />
    <description box="F" iof-2004-ref="10.1" />
    <punch-pattern size="7">
0001000
0000000
0000000
0001000
0000000
0000000
0001000
</punch-pattern>
  </control>
  <control id="76" kind="normal">
    <code>76</code>
    <location x="5.6" y="-5.7" />
    <description box="D" iof-2004-ref="3.4" />
    <description box="E" iof-2004-ref="5.2" />
    <description box="F" iof-2004-ref="10.1" />
    <punch-pattern size="7">
1000001
0000000
0000000
1001001
0000000
0000000
0001000
</punch-pattern>
  </control>
  <control id="77" kind="normal">
    <code>77</code>
    <location x="74.7" y="19.2" />
    <description box="D" iof-2004-ref="3.5" />
    <description box="E" iof-2004-ref="3.5" />
    <description box="F" iof-2004-ref="10.2" />
    <punch-pattern size="7">
0001001
0000000
0000000
1001000
0000000
0000000
1000000
</punch-pattern>
  </control>
  <control id="78" kind="normal">
    <code>78</code>
    <location x="93.9" y="28.1" />
    <description box="D" iof-2004-ref="5.11" />
    <description box="G" iof-2004-ref="11.5SW" />
    <punch-pattern size="7">
1000001
0000000
0000000
0010100
0000000
0000000
1000001
</punch-pattern>
  </control>
  <control id="79" kind="normal">
    <code>79</code>
    <location x="119.8" y="8.6" />
    <description box="D" iof-2004-ref="4.8" />
    <description box="G" iof-2004-ref="11.2W" />
    <punch-pattern size="7">
0000000
0100100
0000000
0000000
0100100
0000000
0000000
</punch-pattern>
  </control>
  <control id="80" kind="normal">
    <code>80</code>
    <location x="41.9" y="6.8" />
    <description box="D" iof-2004-ref="4.3" />
    <description box="G" iof-2004-ref="11.4S" />
    <punch-pattern size="7">
0001000
0000000
0000000
0001000
0000000
0000000
0001000
</punch-pattern>
  </control>
  <control id="81" kind="start">
    <location x="85.84255" y="4.935439" />
  </control>
  <control id="82" kind="finish">
    <location x="54.7981033" y="45.38729" />
    <description box="all" iof-2004-ref="14.3" />
  </control>
  <control id="83" kind="crossing-point" orientation="0">
    <location x="25.6913948" y="-30.8301849" />
    <description box="all" iof-2004-ref="13.3" />
  </control>
  <control id="84" kind="crossing-point" orientation="0">
    <location x="1.63871968" y="-11.8117847" />
    <description box="all" iof-2004-ref="13.3" />
  </control>
  <control id="85" kind="crossing-point" orientation="0">
    <location x="41.6332855" y="52.2354622" />
    <description box="all" iof-2004-ref="13.3" />
  </control>
  <course id="1" kind="normal" order="1">
    <name>Course 1</name>
    <labels label-kind="sequence" />
    <first course-control="101" />
    <options print-scale="10000" description-kind="symbols-and-text" />
  </course>
  <course id="7" kind="normal" order="2">
    <name>Course 2</name>
    <secondary-title>  M20 , F20  , F35|F40,F50,|F65</secondary-title>
    <labels label-kind="sequence" />
    <first course-control="621" />
    <options print-scale="7500" climb="157" description-kind="text" />
  </course>
  <course id="8" kind="normal" order="4">
    <name>NoStart</name>
    <secondary-title>M20, F20, F35</secondary-title>
    <labels label-kind="sequence" />
    <first course-control="629" />
    <options print-scale="10000" climb="35" description-kind="symbols" />
  </course>
  <course id="9" kind="normal" order="3">
    <name>NoFinish</name>
    <labels label-kind="sequence" />
    <first course-control="632" />
    <options print-scale="10000" description-kind="symbols" />
  </course>
  <course id="10" kind="score" order="5">
    <name>Score</name>
    <secondary-title>WIOL Elementary, Public Beginner</secondary-title>
    <labels label-kind="code" />
    <first course-control="636" />
    <options print-scale="10000" climb="40" score-column="A" description-kind="symbols" />
  </course>
  <course id="11" kind="score" order="6">
    <name>ScoreNoPoints</name>
    <labels label-kind="code" />
    <first course-control="656" />
    <options print-scale="15000" score-column="A" description-kind="symbols" />
  </course>
  <course-control id="101" control="1">
    <next course-control="102" />
  </course-control>
  <course-control id="102" control="73">
    <next course-control="103" />
  </course-control>
  <course-control id="103" control="44">
    <next course-control="104" />
  </course-control>
  <course-control id="104" control="49">
    <next course-control="105" />
  </course-control>
  <course-control id="105" control="53">
    <next course-control="106" />
  </course-control>
  <course-control id="106" control="58">
    <next course-control="107" />
  </course-control>
  <course-control id="107" control="76">
    <next course-control="108" />
  </course-control>
  <course-control id="108" control="70">
    <next course-control="109" />
  </course-control>
  <course-control id="109" control="71">
    <next course-control="110" />
  </course-control>
  <course-control id="110" control="55">
    <next course-control="111" />
  </course-control>
  <course-control id="111" control="38">
    <next course-control="112" />
  </course-control>
  <course-control id="112" control="2" />
  <course-control id="628" control="85">
    <next course-control="625" />
  </course-control>
  <course-control id="627" control="84">
    <next course-control="623" />
  </course-control>
  <course-control id="626" control="83">
    <next course-control="627" />
  </course-control>
  <course-control id="625" control="82" />
  <course-control id="624" control="48">
    <next course-control="628" />
  </course-control>
  <course-control id="623" control="72">
    <next course-control="624" />
  </course-control>
  <course-control id="622" control="59">
    <next course-control="626" />
  </course-control>
  <course-control id="621" control="81">
    <next course-control="622" />
  </course-control>
  <course-control id="629" control="80">
    <next course-control="630" />
  </course-control>
  <course-control id="630" control="50">
    <next course-control="631" />
  </course-control>
  <course-control id="631" control="82" />
  <course-control id="632" control="81">
    <next course-control="634" />
  </course-control>
  <course-control id="634" control="58">
    <next course-control="635" />
  </course-control>
  <course-control id="635" control="50" />
  <course-control id="636" control="1">
    <next course-control="638" />
  </course-control>
  <course-control id="638" control="72" points="10">
    <next course-control="639" />
  </course-control>
  <course-control id="639" control="58" points="30">
    <next course-control="640" />
  </course-control>
  <course-control id="640" control="71" points="20">
    <next course-control="641" />
  </course-control>
  <course-control id="641" control="45" points="5">
    <next course-control="642" />
  </course-control>
  <course-control id="642" control="75" points="15">
    <next course-control="643" />
  </course-control>
  <course-control id="643" control="56" points="30">
    <next course-control="644" />
  </course-control>
  <course-control id="644" control="78" points="10">
    <next course-control="645" />
  </course-control>
  <course-control id="645" control="35" points="5">
    <next course-control="646" />
  </course-control>
  <course-control id="646" control="37" points="40">
    <next course-control="647" />
  </course-control>
  <course-control id="647" control="50" points="5">
    <next course-control="648" />
  </course-control>
  <course-control id="648" control="47" points="20">
    <next course-control="649" />
  </course-control>
  <course-control id="649" control="48" points="10">
    <next course-control="650" />
  </course-control>
  <course-control id="650" control="41" points="15">
    <next course-control="651" />
  </course-control>
  <course-control id="651" control="42" points="40">
    <next course-control="652" />
  </course-control>
  <course-control id="652" control="38" points="5">
    <next course-control="653" />
  </course-control>
  <course-control id="653" control="55" points="5">
    <next course-control="654" />
  </course-control>
  <course-control id="654" control="2" />
  <course-control id="656" control="81">
    <next course-control="660" />
  </course-control>
  <course-control id="658" control="82" />
  <course-control id="660" control="50">
    <next course-control="661" />
  </course-control>
  <course-control id="661" control="80">
    <next course-control="662" />
  </course-control>
  <course-control id="662" control="73">
    <next course-control="663" />
  </course-control>
  <course-control id="663" control="56">
    <next course-control="664" />
  </course-control>
  <course-control id="664" control="43">
    <next course-control="665" />
  </course-control>
  <course-control id="665" control="42">
    <next course-control="666" />
  </course-control>
  <course-control id="666" control="54">
    <next course-control="667" />
  </course-control>
  <course-control id="667" control="57">
    <next course-control="658" />
  </course-control>
  <leg id="1" start-control="73" end-control="44">
    <bends>
      <location x="53.27524" y="-30.3053684" />
    </bends>
  </leg>
  <leg id="2" start-control="72" end-control="48">
    <gaps>
      <gap start="8.379353" length="16.2227345" />
    </gaps>
  </leg>
  <leg id="3" start-control="81" end-control="59">
    <bends>
      <location x="66.80469" y="0.214555383" />
      <location x="69.60151" y="-24.39749" />
    </bends>
  </leg>
  <special-object id="3" kind="descriptions">
    <location x="152.2823" y="42.0981445" />
    <location x="158.2241" y="42.0981445" />
    <courses all="true" />
  </special-object>
  <special-object id="4" kind="water">
    <location x="72.67218" y="10.4624271" />
    <courses all="true" />
  </special-object>
</course-scribe-event>