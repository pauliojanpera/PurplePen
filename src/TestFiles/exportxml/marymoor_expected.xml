<?xml version="1.0" encoding="utf-8"?>
<CourseData>
  <IOFVersion version="2.0.3" />
  <ModifyDate>
    <Date>2013-05-28</Date>
    <Clock>13:04</Clock>
  </ModifyDate>
  <Map>
    <Scale>10000</Scale>
    <MapPosition x="-29.5" y="86.7" />
  </Map>
  <StartPoint>
    <StartPointCode>STA1</StartPointCode>
    <MapPosition x="56.8" y="-8.7" />
  </StartPoint>
  <StartPoint>
    <StartPointCode>STA2</StartPointCode>
    <MapPosition x="85.84" y="4.94" />
  </StartPoint>
  <Control>
    <ControlCode>35</ControlCode>
    <MapPosition x="115.6" y="-11.8" />
  </Control>
  <Control>
    <ControlCode>36</ControlCode>
    <MapPosition x="128.4" y="6.1" />
  </Control>
  <Control>
    <ControlCode>37</ControlCode>
    <MapPosition x="121.8" y="-23" />
  </Control>
  <Control>
    <ControlCode>38</ControlCode>
    <MapPosition x="50.3" y="2.9" />
  </Control>
  <Control>
    <ControlCode>39</ControlCode>
    <MapPosition x="129.2" y="37.7" />
  </Control>
  <Control>
    <ControlCode>40</ControlCode>
    <MapPosition x="124.2" y="29.5" />
  </Control>
  <Control>
    <ControlCode>41</ControlCode>
    <MapPosition x="28" y="6.2" />
  </Control>
  <Control>
    <ControlCode>42</ControlCode>
    <MapPosition x="38.4" y="-14" />
  </Control>
  <Control>
    <ControlCode>43</ControlCode>
    <MapPosition x="84" y="34.5" />
  </Control>
  <Control>
    <ControlCode>44</ControlCode>
    <MapPosition x="37.6" y="-22.8" />
  </Control>
  <Control>
    <ControlCode>45</ControlCode>
    <MapPosition x="55.9" y="25.6" />
  </Control>
  <Control>
    <ControlCode>46</ControlCode>
    <MapPosition x="-9.2" y="8.3" />
  </Control>
  <Control>
    <ControlCode>47</ControlCode>
    <MapPosition x="2.51" y="31.32" />
  </Control>
  <Control>
    <ControlCode>48</ControlCode>
    <MapPosition x="21.5" y="40.2" />
  </Control>
  <Control>
    <ControlCode>49</ControlCode>
    <MapPosition x="22" y="-20.8" />
  </Control>
  <Control>
    <ControlCode>50</ControlCode>
    <MapPosition x="45.2" y="22.8" />
  </Control>
  <Control>
    <ControlCode>51</ControlCode>
    <MapPosition x="16.4" y="-11.9" />
  </Control>
  <Control>
    <ControlCode>52</ControlCode>
    <MapPosition x="43" y="-11.1" />
  </Control>
  <Control>
    <ControlCode>53</ControlCode>
    <MapPosition x="25.8" y="-10.5" />
  </Control>
  <Control>
    <ControlCode>54</ControlCode>
    <MapPosition x="53.7" y="8.2" />
  </Control>
  <Control>
    <ControlCode>55</ControlCode>
    <MapPosition x="51.2" y="10.5" />
  </Control>
  <Control>
    <ControlCode>56</ControlCode>
    <MapPosition x="81.3" y="24" />
  </Control>
  <Control>
    <ControlCode>57</ControlCode>
    <MapPosition x="90" y="19.3" />
  </Control>
  <Control>
    <ControlCode>58</ControlCode>
    <MapPosition x="20.3" y="-4" />
  </Control>
  <Control>
    <ControlCode>59</ControlCode>
    <MapPosition x="51.2" y="-27" />
  </Control>
  <Control>
    <ControlCode>70</ControlCode>
    <MapPosition x="21.3" y="11.8" />
  </Control>
  <Control>
    <ControlCode>71</ControlCode>
    <MapPosition x="40.1" y="17.4" />
  </Control>
  <Control>
    <ControlCode>72</ControlCode>
    <MapPosition x="-0.7" y="10.3" />
  </Control>
  <Control>
    <ControlCode>73</ControlCode>
    <MapPosition x="57.4" y="-13.8" />
  </Control>
  <Control>
    <ControlCode>74</ControlCode>
    <MapPosition x="61.2" y="-18.8" />
  </Control>
  <Control>
    <ControlCode>75</ControlCode>
    <MapPosition x="72" y="27" />
  </Control>
  <Control>
    <ControlCode>76</ControlCode>
    <MapPosition x="5.6" y="-5.7" />
  </Control>
  <Control>
    <ControlCode>77</ControlCode>
    <MapPosition x="74.7" y="19.2" />
  </Control>
  <Control>
    <ControlCode>78</ControlCode>
    <MapPosition x="93.9" y="28.1" />
  </Control>
  <Control>
    <ControlCode>79</ControlCode>
    <MapPosition x="119.8" y="8.6" />
  </Control>
  <Control>
    <ControlCode>80</ControlCode>
    <MapPosition x="41.9" y="6.8" />
  </Control>
  <FinishPoint>
    <FinishPointCode>FIN1</FinishPointCode>
    <MapPosition x="53.2" y="-2.8" />
  </FinishPoint>
  <FinishPoint>
    <FinishPointCode>FIN2</FinishPointCode>
    <MapPosition x="54.8" y="45.39" />
  </FinishPoint>
  <Course>
    <CourseName>Course 1</CourseName>
    <CourseId>0</CourseId>
    <CourseVariation>
      <CourseVariationId>0</CourseVariationId>
      <CourseLength>1600</CourseLength>
      <StartPointCode>STA1</StartPointCode>
      <CourseControl>
        <Sequence>1</Sequence>
        <ControlCode>73</ControlCode>
        <LegLength>51</LegLength>
      </CourseControl>
      <CourseControl>
        <Sequence>2</Sequence>
        <ControlCode>44</ControlCode>
        <LegLength>344</LegLength>
      </CourseControl>
      <CourseControl>
        <Sequence>3</Sequence>
        <ControlCode>49</ControlCode>
        <LegLength>157</LegLength>
      </CourseControl>
      <CourseControl>
        <Sequence>4</Sequence>
        <ControlCode>53</ControlCode>
        <LegLength>110</LegLength>
      </CourseControl>
      <CourseControl>
        <Sequence>5</Sequence>
        <ControlCode>58</ControlCode>
        <LegLength>85</LegLength>
      </CourseControl>
      <CourseControl>
        <Sequence>6</Sequence>
        <ControlCode>76</ControlCode>
        <LegLength>148</LegLength>
      </CourseControl>
      <CourseControl>
        <Sequence>7</Sequence>
        <ControlCode>70</ControlCode>
        <LegLength>235</LegLength>
      </CourseControl>
      <CourseControl>
        <Sequence>8</Sequence>
        <ControlCode>71</ControlCode>
        <LegLength>196</LegLength>
      </CourseControl>
      <CourseControl>
        <Sequence>9</Sequence>
        <ControlCode>55</ControlCode>
        <LegLength>131</LegLength>
      </CourseControl>
      <CourseControl>
        <Sequence>10</Sequence>
        <ControlCode>38</ControlCode>
        <LegLength>77</LegLength>
      </CourseControl>
      <FinishPointCode>FIN1</FinishPointCode>
      <DistanceToFinish>64</DistanceToFinish>
    </CourseVariation>
  </Course>
  <Course>
    <CourseName>Course 2</CourseName>
    <CourseId>1</CourseId>
    <ClassShortName>M20</ClassShortName>
    <ClassShortName>F20</ClassShortName>
    <ClassShortName>F35</ClassShortName>
    <ClassShortName>F40</ClassShortName>
    <ClassShortName>F50</ClassShortName>
    <ClassShortName>F65</ClassShortName>
    <CourseVariation>
      <CourseVariationId>0</CourseVariationId>
      <CourseLength>2200</CourseLength>
      <CourseClimb>155</CourseClimb>
      <StartPointCode>STA2</StartPointCode>
      <CourseControl>
        <Sequence>1</Sequence>
        <ControlCode>59</ControlCode>
        <LegLength>630</LegLength>
      </CourseControl>
      <CourseControl>
        <Sequence>2</Sequence>
        <ControlCode>72</ControlCode>
        <LegLength>787</LegLength>
      </CourseControl>
      <CourseControl>
        <Sequence>3</Sequence>
        <ControlCode>48</ControlCode>
        <LegLength>372</LegLength>
      </CourseControl>
      <FinishPointCode>FIN2</FinishPointCode>
      <DistanceToFinish>383</DistanceToFinish>
    </CourseVariation>
  </Course>
  <Course>
    <CourseName>Score</CourseName>
    <CourseId>2</CourseId>
    <ClassShortName>WIOL Elementary</ClassShortName>
    <ClassShortName>Public Beginner</ClassShortName>
    <CourseVariation>
      <CourseVariationId>0</CourseVariationId>
      <StartPointCode>STA1</StartPointCode>
      <CourseControl>
        <Sequence>1</Sequence>
        <ControlCode>35</ControlCode>
        <ScoreOPoints>5</ScoreOPoints>
      </CourseControl>
      <CourseControl>
        <Sequence>2</Sequence>
        <ControlCode>38</ControlCode>
        <ScoreOPoints>5</ScoreOPoints>
      </CourseControl>
      <CourseControl>
        <Sequence>3</Sequence>
        <ControlCode>45</ControlCode>
        <ScoreOPoints>5</ScoreOPoints>
      </CourseControl>
      <CourseControl>
        <Sequence>4</Sequence>
        <ControlCode>50</ControlCode>
        <ScoreOPoints>5</ScoreOPoints>
      </CourseControl>
      <CourseControl>
        <Sequence>5</Sequence>
        <ControlCode>55</ControlCode>
        <ScoreOPoints>5</ScoreOPoints>
      </CourseControl>
      <CourseControl>
        <Sequence>6</Sequence>
        <ControlCode>48</ControlCode>
        <ScoreOPoints>10</ScoreOPoints>
      </CourseControl>
      <CourseControl>
        <Sequence>7</Sequence>
        <ControlCode>72</ControlCode>
        <ScoreOPoints>10</ScoreOPoints>
      </CourseControl>
      <CourseControl>
        <Sequence>8</Sequence>
        <ControlCode>78</ControlCode>
        <ScoreOPoints>10</ScoreOPoints>
      </CourseControl>
      <CourseControl>
        <Sequence>9</Sequence>
        <ControlCode>41</ControlCode>
        <ScoreOPoints>15</ScoreOPoints>
      </CourseControl>
      <CourseControl>
        <Sequence>10</Sequence>
        <ControlCode>75</ControlCode>
        <ScoreOPoints>15</ScoreOPoints>
      </CourseControl>
      <CourseControl>
        <Sequence>11</Sequence>
        <ControlCode>47</ControlCode>
        <ScoreOPoints>20</ScoreOPoints>
      </CourseControl>
      <CourseControl>
        <Sequence>12</Sequence>
        <ControlCode>71</ControlCode>
        <ScoreOPoints>20</ScoreOPoints>
      </CourseControl>
      <CourseControl>
        <Sequence>13</Sequence>
        <ControlCode>56</ControlCode>
        <ScoreOPoints>30</ScoreOPoints>
      </CourseControl>
      <CourseControl>
        <Sequence>14</Sequence>
        <ControlCode>58</ControlCode>
        <ScoreOPoints>30</ScoreOPoints>
      </CourseControl>
      <CourseControl>
        <Sequence>15</Sequence>
        <ControlCode>37</ControlCode>
        <ScoreOPoints>40</ScoreOPoints>
      </CourseControl>
      <CourseControl>
        <Sequence>16</Sequence>
        <ControlCode>42</ControlCode>
        <ScoreOPoints>40</ScoreOPoints>
      </CourseControl>
      <FinishPointCode>FIN1</FinishPointCode>
    </CourseVariation>
  </Course>
  <Course>
    <CourseName>ScoreNoPoints</CourseName>
    <CourseId>3</CourseId>
    <CourseVariation>
      <CourseVariationId>0</CourseVariationId>
      <StartPointCode>STA2</StartPointCode>
      <CourseControl>
        <Sequence>1</Sequence>
        <ControlCode>42</ControlCode>
      </CourseControl>
      <CourseControl>
        <Sequence>2</Sequence>
        <ControlCode>43</ControlCode>
      </CourseControl>
      <CourseControl>
        <Sequence>3</Sequence>
        <ControlCode>50</ControlCode>
      </CourseControl>
      <CourseControl>
        <Sequence>4</Sequence>
        <ControlCode>54</ControlCode>
      </CourseControl>
      <CourseControl>
        <Sequence>5</Sequence>
        <ControlCode>56</ControlCode>
      </CourseControl>
      <CourseControl>
        <Sequence>6</Sequence>
        <ControlCode>57</ControlCode>
      </CourseControl>
      <CourseControl>
        <Sequence>7</Sequence>
        <ControlCode>73</ControlCode>
      </CourseControl>
      <CourseControl>
        <Sequence>8</Sequence>
        <ControlCode>80</ControlCode>
      </CourseControl>
      <FinishPointCode>FIN2</FinishPointCode>
    </CourseVariation>
  </Course>
</CourseData>